### ISSUE TITLE
Unit cost of LqdMethane seems abnormally high 

### ISSUE THREAD

**Issue by jmburbach** [BobPalmer/CommunityResourcePack#259]
Unit cost of LqdMethane is set to 0.45, which is quite high compared to other liquid fuels. For example LqdHydrogen is only 0.03675, and Kerosene is 0.041.

This causes the cost of a tank of LqdMethane to be extremely high. A sample tank, set to LqdHydrogen, had a cost of only 1,338....with LqdMethane cost was an incredible 15,552!

This maybe was a typo, where 0.45 should have been 0.045.

**Comment by sswelm** [comment]
I agree it should be balanced against the cos of LqdHydrogen. Technically it should be a lot cheaper as it's far easier to store and create.
